### PR TITLE
添加 mindspore serving 支持

### PR DESCRIPTION
**Pull request by Hsiayukoo**
Thank you for your contribution to the MindOCR repo.
Before submitting this PR, please make sure:

- [x] You have read the [Contributing Guidelines on pull requests](https://github.com/mindspore-lab/mindcv/blob/main/CONTRIBUTING.md)
- [x] Your code builds clean without any errors or warnings
- [x] You are using approved terminology
- [x] You have added unit tests

## Motivation

添加 mindspore serving 支持，提供几行命令即可使用 mindspore serving 部署 mindocr 的相关模型的功能。

## Test Plan

见附件测试结果

[模型测试结果.pdf](https://github.com/user-attachments/files/17597128/default.pdf)

## Related Issues and PRs

(Is this PR part of a group of changes? Link the other relevant PRs and Issues here. Use https://help.github.com/en/articles/closing-issues-using-keywords for help on GitHub syntax)

### CODE REVIEW

File: deploy/ocr_serving/package_utils/package_helper.py [hunk 1]
```diff
@@ -0,0 +1,294 @@
...
+            self.copy_custom_mindir_file()
+
+    def get_official_mindir_file(self):
+        """
+        download official ckpt file and convert it to mindir file
+        Returns:
+        """
+        # 1. get ckpt file from official link
+        response = requests.get(self.target_config_yaml["ckpt_link"])
+        if math.floor(response.status_code / 100) != 2:
```
Comment (wxsIcey) [inline deploy/ocr_serving/package_utils/package_helper.py]: 魔鬼数字定义成常量

File: deploy/ocr_serving/test/serving_client.py [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+            image_files.append(image_file)
+    for image_file in image_files:
+        with open(image_file, "rb") as fp:
+            images_buffer.append(fp.read())
+    return images_buffer, image_files
+
+
+def run_restful_classify_top1(model_name, restful_address):
+    """RESTful Client for servable lenet and method classify_top1"""
+    print("run_restful_classify_top1-----------")
```
Comment (wxsIcey) [inline deploy/ocr_serving/test/serving_client.py:22]: 尽量以日志方式打印

Comment (Hsiayukoo) [reply deploy/ocr_serving/test/serving_client.py:22]: 这是测试脚本

File: deploy/ocr_serving/package_utils/package_helper.py (hunk 1 above)
Comment (Hsiayukoo) [reply deploy/ocr_serving/package_utils/package_helper.py]: 已修改

**Review by wxsIcey — APPROVED** [review]

**Pull request merged**